tools/tools.go

```diff
@@ -0,0 +1,9 @@
+//go:build tools
+// +build tools
+package tools
+
+import (
+	_ "github.com/ahmetb/gen-crd-api-reference-docs"
```

Comment:
What is this being used for? I don't see any generated docs?

Reply:
Removed it.